server/controllers/manage/lostBedsController.ts

```diff
@@ -36,6 +36,7 @@ export default class LostBedsController {
 
       const lostBed: NewLostBed = {
         ...req.body.lostBed,
+        bedId,
```

Comment:
Is this intentionally its own commit?

Reply:
Yeah, this was a bug introduced by a previous feature